src/shared/components/search-filters/search-filters.component.ts

```diff
@@ -209,4 +211,10 @@ export class SearchFiltersComponent implements OnInit {
       this.onCaseTypeIdChange();
     }
   }
+
+  private getCaseFields() {
```

Comment:
Return type of `: void`?

Reply:
I thought we were making lifecycle hooks public and with a return type of void (like 34 other onInit lifecycle hooks in the lib)?

Reply:
This isn't a lifecycle hook - it's a new method that you've added, isn't it? It should have a return type. 🤷‍♂️